src/main-process/parse-command-line.js

```diff
@@ -177,7 +193,25 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
-function normalizeDriveLetterName (filePath) {
+const readProjectSettingsSync = (filepath, executedFrom) => {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
+    const contents = CSON.readFileSync(readPath)
+    if (contents.paths && contents.config) {
+      return contents
+    }
+    throw new Error()
```

Comment:
throwing an empty error and then immediately catching it is kinda weird. you can just move the later throw out of the catch block:

```js
try {
    const contents = CSON.readFileSync(readPath)
    if (contents.paths && contents.config) return contents
} catch (err) {}
throw new Error(...)
```

or maybe even have two separate errors